js/songs/index.controller.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+(function(){
+  angular
+  .module("songs")
+  .controller("SongIndexController", [
+    "SongFactory",
+    SongIndexControllerFunction
+  ]);
+
+  function SongIndexControllerFunction(SongFactory){
+    this.songs = SongFactory.query();
```

Comment:
As I said in my earlier comment, this can't be done without defining and linking to a factory file.